bp_fe/src/include/bp_fe_itlb_pkg.vh

```diff
@@ -0,0 +1,5 @@
+package itlb_pkg;
+
+localparam bp_ppn_width_gp = 44;
```

Comment:
Should add the `include itlb headers in this package and then import the package where needed, rather than `including files individually

Reply:
updated